test-suite/fdheston.cpp

```diff
@@ -641,8 +641,8 @@ void FdHestonTest::testFdmHestonIntradayPricing() {
     option.setPricingEngine(fdm);
 
     const Real gammaExpected[] = {
-        1.46702, 1.54638, 1.64018, 1.75343, 1.89393,
-        2.0747, 2.31959, 2.67844, 3.28041, 4.63922  };
+        1.46757, 1.54696, 1.6408, 1.75409, 1.89464,
+        2.07548, 2.32046, 2.67944, 3.28164, 4.64096  };
```

Comment:
a while ago the spot mesher in FdHestonVanillaEngine was changed in line 77 from

                      Null<Real>(), Null<Real>(), 0.0001, 1.5,
towards
                      Null<Real>(), Null<Real>(), 0.0001, 2.0,

to improve the price quality of deep OTM options. This has also slightly changed the Gamma values in this test because the option is only hours before expiry but ATM. Since this test is only running with --enable-intraday it was overseen back then.